agent/config/types.go

```diff
@@ -35,12 +35,17 @@ type Cloud struct {
 	MQTT   MQTTConfig  `mapstructure:"mqtt"`
 }
 
+type Opentelemetry struct {
+	Scrape bool `mapstructure:"scrape"`
```

Comment:
Sorry I should have been more specific on the naming - can you name this "Enabled" instead of "Scrape" and adjust the rest of the code changes here to change that word to please?